### DISCUSSION TITLE
custom decade range

### DISCUSSION THREAD

**Discussion by ShlomyYosef** [wojtekmaj/react-calendar#779] [Q&A]
Hello,
When the user goes to the decade view, there are only 10 years visible and there are two tiles left unoccupied. Is there a way to show 12 years?
also, there is a way to change the years?
I know we can use formatYear or tileContent, but then when I will click on the diary I will still get the original date.
for example, I want to show 2014-2025.

**Comment by kaungmyathein12** [comment]
hello @ShlomyYosef we can use `showNeighboringDecade` and `showNeighboringCentury`
